### PR TITLE
Add stellar cyber partner

### PR DESCRIPTION
**Pull request by sjones100**
### Description
Added Stellar Cyber as partner
 
### Issues Resolved
N/A

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _partners/stellar-cyber.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+name: Stellar Cyber
+logo: '/assets/media/partners/stellar-cyber.png'
```
Comment (krisfreedain) [inline _partners/stellar-cyber.md:3]: Hello @sjones100 - small edit needed - the logo png needs to be square

**Review by krisfreedain — CHANGES_REQUESTED** [review]
@sjones100 - small change needed

**Comment by krisfreedain** [comment]
Hello @sjones100 - changes look good. However, you need to fix the DCO on your commit for this to progress forward https://github.com/opensearch-project/project-website/pull/814/checks?check_run_id=6475852461

**Comment by sjones100** [comment]
@krisfreedain apologies, just rebased and fixed, looks like DCO check has passed now

**Comment by krisfreedain** [comment]
thanks @sjones100 - glad to add you to the partners page! https://opensearch.org/partners/

**Pull request merged**